src/servers/src/http.rs

```diff
@@ -127,6 +127,8 @@ pub struct HttpOptions {
     pub disable_dashboard: bool,
 
     pub body_limit: ReadableSize,
+
+    pub is_strict_mode: bool,
```

Comment:
since the field has been updated, maybe it make sense to update the corresponding arguments from `strict_mode` to `is_strict_mode`.